spec/factories/kudos.rb

```diff
@@ -6,5 +6,6 @@
     content { 'content' }
     giver factory: :employee
     receiver factory: :employee
+    company_value factory: :company_value
```

Comment:
You do not have to specify the factory here as well, FactoryBot will get it out of the name of the class, since they are identical 

```suggestion
    company_value 
```

Reply:
ok